### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by TravisTroyer**
[Exporter.Instana] Release 1.1.0.  Fixes [747](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/issues/747).

## Changes
- Adds `InstanaExporterEventSource` to provide for error logging.
- Adds exception-handling to `Transport` with logging via `InstanaExporterEventSource`.
- Fixes `Transport` request buffering to remove potential for exceeding underlying array capacity.
- Fixes `Transport` to remove potential for lost spans due to buffer length.

### CODE REVIEW

**Comment by TravisTroyer** [comment]
@open-telemetry/dotnet-contrib-maintainers

File: src/OpenTelemetry.Exporter.Instana/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Changelog
 
-## Unreleased
+## 1.1.0
```
Comment (utpilla) [inline src/OpenTelemetry.Exporter.Instana/CHANGELOG.md:3]: I think we should probably release this version as 1.0.2 as these changes are mainly bug fixes.

Comment (TravisTroyer) [reply src/OpenTelemetry.Exporter.Instana/CHANGELOG.md]: That works for me.  I was on the fence as to whether the new exception logging was really a minor feature.

**Review by utpilla — APPROVED** [review]

**Comment by utpilla** [comment]
The package is published: https://www.nuget.org/packages/OpenTelemetry.Exporter.Instana/1.0.2

**Pull request merged**